app/src/main/res/layout/view_debug_drawer_configure_environment.xml

```diff
@@ -39,4 +39,15 @@
         android:text="Enable auto update"
         tools:ignore="HardcodedText" />
 
+    <androidx.appcompat.widget.SwitchCompat
```

Comment:
Why `Switch` is not okay here?

Reply:
I think it is ok... it was just copy paste here... but I agree we need to get rid from this not needed androidX prefixes